contracts/middle-layer/resource-mirror/AdditionalBucketHub.sol

```diff
@@ -72,49 +72,144 @@ contract AdditionalBucketHub is BucketStorage, GnfdAccessControl {
      * @param synPkg Package containing information of the bucket to be created
      */
     function createBucket(CreateBucketSynPackage memory synPkg) external payable returns (bool) {
+        (uint8 _channelId, bytes memory _msgBytes, uint256 _relayFee, uint256 _ackRelayFee, ) = _prepareCreateBucket(
```

Comment:
I think we dont have to return channel id _prepareCreateBucket, because we all know it is `BUCKET_CHANNEL_ID`, use it directly

Reply:
so is `relayFee` and `minAckRelayFee`